packages/backend/src/peripherals/multicall/codecs.ts

```diff
@@ -1,15 +1,18 @@
 import { Bytes, EthereumAddress } from '@l2beat/shared-pure'
 import { BigNumber, utils } from 'ethers'
 
-import { ETHEREUM_BALANCE_ENCODING } from '../../modules/tvl/balances/BalanceProvider'
+import {
+  ETHEREUM_MULTICALL_V1_ADDRESS,
+  ETHEREUM_MULTICALL_V1_BLOCK,
+} from './MulticallConfig'
 import {
   ERC20MulticallCodec,
   MulticallRequest,
   NativeAssetMulticallCodec,
 } from './types'
 
 export const nativeAssetCodec: NativeAssetMulticallCodec = {
-  sinceBlock: ETHEREUM_BALANCE_ENCODING.sinceBlock,
+  sinceBlock: ETHEREUM_MULTICALL_V1_BLOCK,
```

Comment:
This will be phased out in the PR "Fix multicall issues"

Reply:
noted